routers/web/feed/profile.go

```diff
@@ -23,31 +23,34 @@ func RetrieveFeeds(ctx *context.Context, options models.GetFeedsOptions) []*mode
 		return nil
 	}
 
-	userCache := map[int64]*user_model.User{options.RequestedUser.ID: options.RequestedUser}
-	if ctx.User != nil {
-		userCache[ctx.User.ID] = ctx.User
-	}
-	for _, act := range actions {
-		if act.ActUser != nil {
-			userCache[act.ActUserID] = act.ActUser
+	// TODO: move load repoOwner of act.Repo into models.GetFeeds->loadAttributes()
+	{
```

Comment:
Why this?

Reply:
note to refactor - but that would make this pull to large, next pull will follow for sure ... to round up and make repo rss posible (backend should already support it but its not exported & not tested)

Reply:
I was meaning the placing of this section of code within a {} block

Reply:
-> #19055

or to be specific: https://github.com/go-gitea/gitea/pull/19055/commits/ed80cfc5cc38e7d14e1000e00e7c4047fb851013